torch/export/__init__.py

```diff
@@ -1020,9 +1042,12 @@ def specify_constraints(x):
 
     """
 
-    from torch._export import export
+    from torch._export import export, export__RC__
 
-    return export(f, args, kwargs, constraints)
+    if constraints is not None:
+        return export(f, args, kwargs, constraints)
+    else:
+        return export__RC__(f, args, kwargs, dynamic_shapes=dynamic_shapes)
```

Comment:
I'll punt on it. Otherwise I'll have to check annotations right here. Given we will deprecate or even remove `constraints` soon, I doubt this will come up in practice.